projects/layout/src/lib/components/configurable-layout/configurable-layout.component.ts

```diff
@@ -95,27 +93,31 @@ export class NgxConfigurableLayoutComponent
 	 * Providing both a `control` and the `keys` input will result in the control overriding
 	 * the input.
 	 */
-	@Input() public set keys(keys: string[]) {
+	@Input() public set keys(keys: string[][]) {
 		// Wouter: If no keys are provided, we prevent the patching of the control.
 		if (!Boolean(keys)) {
 			return;
 		}
 		// Wouter: Patch the provided keys onto the control.
 		this.form.patchValue(keys, { emitEvent: false });
 	}
+
 	/**
-	 * The properties to define the grid in which the items should be displayed. The default
-	 * config is `auto` rows and `1` columns.
+	 * Determines how much space each item takes in the row. By default, this is 'fill'
+	 *
+	 * fill - Will make the items take up the available space
+	 * fit-content - Will make the items take up the space of the provided template
+	 * equal - Will make the items take up equal amount of space throughout the grid
+	 *
+	 * @type {NgxConfigurableLayoutItemSizeOption}
```

Comment:
Shouldn't these tsDoc properties be added to every input, then?

Reply:
Nah, forgot to delete it!